src/main/java/seedu/duke/common/Messages.java

```diff
@@ -146,28 +155,36 @@
     public static final String MESSAGE_QUIZ_DELETE_ERROR_NON_NUMBER = ":( OOPS!!! Please indicate in NUMERALS, "
             + "which quiz you'd like to delete!\n";
     public static final String MESSAGE_INVALID_HELP_COMMAND = ":( OOPS!!! Are you trying to take a quiz or add/delete a"
-            + " quiz question? Enter 'help' to check the correct format!\n";
+            + " quiz question? Enter 'help' to check the correct format!";
     public static final String MESSAGE_EMPTY_QUIZ_LIST = "Quiz list is empty. Add some!";
-    public static final String MESSAGE_QUIZ_INDEX_OUT_OF_BOUND = "There is not such a question in your quiz list!";
+    public static final String MESSAGE_QUIZ_INDEX_OUT_OF_BOUND = "There is no such question in your quiz list!";
+    public static final String MESSAGE_QUIZ_QUESTION_NOT_FOUND = "Question not found";
+    public static final String MESSAGE_QUIZ_ANSWER_NOT_FOUND = "Answer not found";
```

Comment:
Shld we write no question found instead of question not found

Reply:
Edited. Thanks for the suggestion.